### ISSUE TITLE
debian_oval: cannot connect behind a proxy

### ISSUE THREAD

**Issue by list4nestor** [nexB/vulnerablecode#556]
Behind a company proxy, we get this when importing debian_oval:
```
$ python manage.py import debian_oval
Importing data from debian_oval
Failed to get updated_advisories: <xml.etree.ElementTree.ElementTree object at 0x7fb2af8fb550> with {'type': 'deb', 'namespace': 'debian', 'qualifiers': {'distro': 'wheezy'}}:
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/aiohttp/connector.py", line 999, in _create_direct_connection
    hosts = await asyncio.shield(host_resolved)
(...)
```

=> Request to support use of proxy like already done in all other cases.

**Comment by Hritik14** [comment]
@list4nestor Please provide with full error log. In case this is a duplicate of #554, try [this](https://subversion.apache.org/faq.html#proxy) or [this](https://widerin.net/blog/tell-svn-client-to-use-a-proxy-server/)

**Comment by tardyp** [comment]
full stacktrace is here:
```
Importing data from debian_oval
Failed to get updated_advisories: <xml.etree.ElementTree.ElementTree object at 0x7fb2af8fb550> with {'type': 'deb', 'namespace': 'debian', 'qualifiers': {'distro': 'wheezy'}}:
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/aiohttp/connector.py", line 999, in _create_direct_connection
    hosts = await asyncio.shield(host_resolved)
  File "/usr/local/lib/python3.8/site-packages/aiohttp/connector.py", line 999, in _create_direct_connection
    hosts = await asyncio.shield(host_resolved)
  File "/usr/local/lib/python3.8/site-packages/aiohttp/connector.py", line 999, in _create_direct_connection
    hosts = await asyncio.shield(host_resolved)
  [Previous line repeated 22 more times]
  File "/usr/local/lib/python3.8/site-packages/aiohttp/connector.py", line 852, in _resolve_host
    await event.wait()  File "/usr/local/lib/python3.8/site-packages/aiohttp/locks.py", line 38, in wait
    raise self._exc
  File "/usr/local/lib/python3.8/site-packages/aiohttp/connector.py", line 865, in _resolve_host
    addrs = await self._resolver.resolve(host, port, family=self._family)
  File "/usr/local/lib/python3.8/site-packages/aiohttp/resolver.py", line 31, in resolve
    infos = await self._loop.getaddrinfo(
  File "/usr/local/lib/python3.8/asyncio/base_events.py", line 825, in getaddrinfo
    return await self.run_in_executor(
  File "/usr/local/lib/python3.8/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/usr/local/lib/python3.8/socket.py", line 918, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno -2] Name or service not known
```

I already fixed similar issues in ca32550c15a9e1327d88b78f351ce05c6653de5b
but it looks like I forgot one of the instance. not sure why because at that time I remember every importers were working

https://github.com/nexB/vulnerablecode/blob/master/vulnerabilities/package_managers.py#L226
